l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -243,22 +274,14 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
         bytes calldata _initData,
```

Comment:
you can just put assetId in this function rather than baseToken address

Reply:
@kelemeno the problem with it is that in multiple places (such as diamond proxy storage) the base token is expected to be an address and not an id